test/unit/specs/hive/basic-store.js

```diff
@@ -0,0 +1,2237 @@
+          await expect(() => store.expireAt(null)).to.throw(errorMsg1)
+          await expect(() => store.expireAt(false)).to.throw(errorMsg1)
+          await expect(() => store.expireAt(true)).to.throw(errorMsg1)
+          await expect(() => store.expireAt(0)).to.throw(errorMsg1)
+          await expect(() => store.expireAt(123)).to.throw(errorMsg1)
+          await expect(() => store.expireAt(() => undefined)).to.throw(errorMsg1)
+          await expect(() => store.expireAt({})).to.throw(errorMsg1)
+
+          const errorMsg2 = 'Expiration time must be a number.'
+
+          await expect(() => store.expireAt('test', undefined)).to.throw(errorMsg2)
+          await expect(() => store.expireAt('test', null)).to.throw(errorMsg2)
+          await expect(() => store.expireAt('test', false)).to.throw(errorMsg2)
+          await expect(() => store.expireAt('test', true)).to.throw(errorMsg2)
+          await expect(() => store.expireAt('test', NaN)).to.throw(errorMsg2)
+          await expect(() => store.expireAt('test', () => undefined)).to.throw(errorMsg2)
+          await expect(() => store.expireAt('test', {})).to.throw(errorMsg2)
+        })
+      })
+
+      describe('Touch', async () => {
+        it('success with single key', async () => {
+          const req1 = prepareMockRequest(fakeResult)
+
+          const result1 = await store.touch('testKey')
+
+          expect(req1).to.deep.include({
+            method : 'PUT',
+            path   : `${storeUrl}/touch`,
+            headers: { 'Content-Type': 'application/json' },
+            body   : ['testKey']
+          })
+
+          expect(result1).to.be.eql(fakeResult)
+        })
+
+        it('success with multi keys', async () => {
+          const req1 = prepareMockRequest(fakeResult)
+
+          const result1 = await store.touch(['testKey1', 'testKey1'])
+
+          expect(req1).to.deep.include({
+            method : 'PUT',
+            path   : `${storeUrl}/touch`,
+            headers: { 'Content-Type': 'application/json' },
+            body   : ['testKey1', 'testKey1']
+          })
+
+          expect(result1).to.be.eql(fakeResult)
+        })
+
+        it('fails when key(s) is invalid', async () => {
+          const errorMsg = 'Key(s) must be provided and must be a string or list of strings.'
+
+          await expect(() => store.touch(undefined)).to.throw(errorMsg)
+          await expect(() => store.touch(null)).to.throw(errorMsg)
+          await expect(() => store.touch(false)).to.throw(errorMsg)
+          await expect(() => store.touch(0)).to.throw(errorMsg)
+          await expect(() => store.touch(true)).to.throw(errorMsg)
+          await expect(() => store.touch(123)).to.throw(errorMsg)
+          await expect(() => store.touch(() => undefined)).to.throw(errorMsg)
+          await expect(() => store.touch({})).to.throw(errorMsg)
+        })
+      })
+    })
+
+    describe('List', () => {
```

Comment:
shouldn't it be in a separate file, please check other groups

Reply:
done